_posts/2017-8-19-A_Beginner's_Guide_To_Bootstrap.md

```diff
@@ -0,0 +1,206 @@
+
+#### Using the bootstrap grid system to create mobile friendly webpages
+
+```html
+<div class="container">
+    <div class="row">
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">First column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Second column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Third column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Fourth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Fifth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Sixth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Seventh column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Eighth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Ninth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Tenth column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Eleventh column group</div>
+        <div class="col-lg-3 col-md-4 col-sm-6 col-xs-12">Twelveth column group</div>
+        
+    </div>
+</div>
+```
+Here we have declared multiple classes (col-lg-3, col-md-4, ..) for div. This ensures that a grid component occupies different number of columns on an extra small screen than a large screen. On a phone, only col-xs-12 will be considered as the class during column assignment to the grid component, while on a large screen, col-lg-3 will be considered. This is how mobile friendly pages can be created using bootstrap.
+
+There is a lot more to grids, we can provide grid offsets, which can be used to leave blank space between different columns. We can also nest rows into columns.
+
+
+### _Conclusion and Further Reading:_
+Bootstrap is one of the easiest webpage development packages. It is a very useful package to accomplish almost any formatting task. I recommend you to read about all the bootstrap components I mentioned like buttons, forms, icons, and modals. When you are comfortable with the topics, start making your own webpages. It takes a great deal of practice to be a good front-end designer. Now, for some inspiration as to what you can create with bootstrap, check out this bootstrap [official link](http://expo.getbootstrap.com/), where you will find some amazing applications of bootstrap tools for creating some mind-blowing websites!
```

Comment:
Nit: does "I recommend you to read" sound better or "I recommend that you read" sound better?